app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -2,16 +2,14 @@
 
 module Bridges
   module Lom
-    # rubocop:disable Metrics/ClassLength
-    # rubocop:disable Metrics/AbcSize
-    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/ClassLength, Metrics/AbcSize, Metrics/MethodLength
     class TasksController < ActionController::API
       OML_SCHEMA_PATH = Rails.root.join('vendor/assets/schemas/lom_1484.12.3-2020/lom.xsd')
 
       def show
         @task = Task.find(params[:id])
 
-        if @task.showable_by?(current_user)
+        if @task.access_level_public? || @task.author?(current_user)
```

Comment:
Using `@task.author?` is more restrictive than `@task.showable_by?`, is it? Consider the case that I am not an author of a given task but got access through a group. In this case, I would still expect to be able to access the LOM data.

Reply:
That's true. I went back to showable_by? to cover this.

Reply:
Thanks, looks good 👍
By the way: A test for this behavior would be nice 😊

Reply:
I added a test.